sdk/core/Azure.Core/src/Shared/Autorest/OperationFinalStateVia.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+namespace Azure.Core
+{
+    internal enum OperationFinalStateVia
+    {
+        AzureAsyncOperation,
+        Location,
+        OriginalUri
```

Comment:
Nit: recommend terminating commas for easier diffs in the future. There was also a private one above but figured that would never change. Maybe this one won't either.